synapse/push/pusherpool.py

```diff
@@ -454,6 +461,13 @@ async def remove_pusher(self, app_id: str, pushkey: str, user_id: str) -> None:
                 app_id, pushkey, user_id
             )
 
+    async def delete_all_pushers_for_user(self, user_id: str) -> None:
+        """Deletes all pushers for a user."""
+        if self._delete_all_pushers_for_user_client is not None:
+            await self._delete_all_pushers_for_user_client(user_id=user_id)
+        else:
+            await self.store.delete_all_pushers_for_user(user_id)
```

Comment:
Might as well make these look similar

```suggestion
            await self.store.delete_all_pushers_for_user(user_id=user_id)
```